tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -132,3 +156,42 @@ async def run_metadata_orchestrator_test_pipeline(
             metadata_pipeline_context.test_report = TestReport(pipeline_context=metadata_pipeline_context, steps_results=[result])
 
     return metadata_pipeline_context.test_report.success
+
+
+async def run_metadata_orchestrator_deploy_pipeline(
+    is_local: bool,
+    git_branch: str,
+    git_revision: str,
+    gha_workflow_run_url: Optional[str],
+    pipeline_start_timestamp: Optional[int],
+    ci_context: Optional[str],
+) -> bool:
+    # Get the DAGSTER_CLOUD_API_TOKEN from the environment
+    dagster_cloud_api_token = os.environ.get("DAGSTER_CLOUD_API_TOKEN")
+    if not dagster_cloud_api_token:
+        raise ValueError("DAGSTER_CLOUD_API_TOKEN not found in environment")
```

Comment:
I think this validation should happen earlier, at the command level.
You can pass the token as a required command option with an `envvar` default to `DAGSTER_CLOUD_API_TOKEN`. If you make the option required and the envvar is missing I think click will raise an error.